components/multimedia/youtube-dl/Makefile

```diff
@@ -0,0 +1,42 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2020 Nona Hansel
+#
+
+BUILD_BITS=		NO_ARCH
+BUILD_STYLE=		setup.py
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		youtube-dl
+COMPONENT_VERSION=	2020.3.8
+COMPONENT_TMP_VERSION=	2020.03.08
+COMPONENT_SUMMARY=	'Command-line program to download videos from YouTube.com and other video sites'
```

Comment:
As PYV rules are not used, COMPONENT_SUMMARY is not transformed twice and ' signs appear in pkg summary. Please, remove ' from this line..

Reply:
Removed, thanks.